app/serializers/api/admin/product_serializer.rb

```diff
@@ -1,13 +1,21 @@
 class Api::Admin::ProductSerializer < ActiveModel::Serializer
   attributes :id, :name, :sku, :variant_unit, :variant_unit_scale, :variant_unit_name, :on_demand, :inherits_properties
 
-  attributes :on_hand, :price, :available_on, :permalink_live, :tax_category_id
+  attributes :on_hand, :price, :available_on, :permalink_live, :tax_category_id, :image_url, :thumb_url
 
   has_one :supplier, key: :producer_id, embed: :id
   has_one :primary_taxon, key: :category_id, embed: :id
   has_many :variants, key: :variants, serializer: Api::Admin::VariantSerializer # embed: ids
   has_one :master, serializer: Api::Admin::VariantSerializer
 
+  def image_url
+    object.images.present? ? object.images.first.attachment.url(:product) : "/assets/noimage/product.png"
```

Comment:
what's the intent of that `noimage` path? What about simply returning an empty string meaning that there is no product image?

Reply:
It's actually the path to a "default" image, in this case a small OFN logo.

Reply:
that's weird. There's no such path on my local repo. I 👃  some nasty magic here.

Reply:
@sauloperez What do you see when you visit a shop with products that don't have images? 

![screenshot from 2018-01-10 22-32-24](https://user-images.githubusercontent.com/9029026/34799076-460e40a8-f656-11e7-9de4-a4a22c58776d.png)

Reply:
Confirmed. I see that picture